plugins/modules/network/routeros/routeros_facts.py

```diff
@@ -68,6 +68,18 @@
   description: The configured hostname of the device
   returned: always
   type: str
+ansible_net_architec:
+  description: The arcitect device
+  returned: always
+  type: str
```

Comment:
```suggestion
  type: str
  version_added: 1.1.0
```
Same for the others.

Reply:
Do I understand correctly that when adding new variables, you need to specify which version they appeared in? In this case, the question is, where are minor and major version changes tracked? In what case do they change? You indicated above ( version_added: 1.1.0) where you got this information? Do I need to specify it in the Routing function?